### ISSUE TITLE
Admin\Register an Operation- Internal User can see Operation Registration information

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1706]
#### Description:
As an internal user, I am able to see an operation registration application, so that I can approve it or request changes 

#### Acceptance Criteria:

Given I am on the operation registration grid #1705  
When I click on view details
Then I am brought to the Operation Registration Details page
- [ ] [Figma page 1](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7861-1407&t=lP4wgjHuBjopAaSN-4)https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=7861-1407&t=lP4wgjHuBjopAaSN-4
- [ ] [Figma page 2](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=10222-477&t=lP4wgjHuBjopAaSN-4)
- [ ][ Figma page 3](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=10222-508&t=lP4wgjHuBjopAaSN-4)
- [ ] [Figma page 4](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=10222-546&t=lP4wgjHuBjopAaSN-4)

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking
